src/pykeen/triples/triples_numeric_literals_factory.py

```diff
@@ -114,3 +115,7 @@ def create_lcwa_instances(self, use_tqdm: Optional[bool] = None) -> MultimodalLC
             numeric_literals=self.numeric_literals,
             literals_to_id=self.literals_to_id,
         )
+
+    def initializer(self, _) -> torch.FloatTensor:
```

Comment:
@mberr was there a reason we're even considering the numeric literals as np? can we just go straight to a torch float tensor?

Reply:
We can do the conversion during initialization of the factory.

Reply:
Maybe we can rename this to `literal_initializer` to be more precise?

Reply:
I think we should wait on merging #245 before really investing any more time on this, since it will all change anyway

Reply:
Oh but we can do this rename, sure

Reply:
Done in 95c3e73d